tgbot/middlewares/throttling.py

```diff
@@ -0,0 +1,85 @@
+from typing import Any, Awaitable, Callable, Dict
+
+from aiogram import BaseMiddleware
+from aiogram.dispatcher.flags import get_flag
+from aiogram.types import Message
+from cachetools import TTLCache
+
+
+class ThrottlingMiddleware(BaseMiddleware):
```

Comment:
You'll need to remove this implementation of throttling middleware, since I'll be adding another one